cocos2d/Support/CCFile_Private.h

```diff
@@ -55,6 +55,13 @@ typedef NSInputStream *(^CCStreamedImageSourceStreamBlock)();
 @end
 
 
+@interface CCFile()
+
+@property (nonatomic, assign, readwrite) BOOL useUIScale;
+
+@end
+
+
 @interface CCFile(Private)
```

Comment:
Oh. That's a really bizarre auto-property synthesis rule. If you put it in a named extension, you have to use the `@synthesize` keyword in the implementation. I'm honestly not sure why Xcode automatically synthesizes properties in anonymous categories. Seems problematic to me.